packages/core-ethereum/src/indexer/index.ts

```diff
@@ -252,13 +252,15 @@ class Indexer extends EventEmitter {
 
     for (const event of confirmedEvents) {
       const eventName = event.event as EventNames
-      if (eventName === ANNOUNCEMENT) {
-        await this.onAnnouncement(event as Event<'Announcement'>, new BN(blockNumber.toPrecision()))
-      } else if (eventName === 'ChannelUpdated') {
-        await this.onChannelUpdated(event as Event<'ChannelUpdated'>)
-      } else {
-        log('skipping event: ', eventName, ' as it isnt recognized')
```

Comment:
would be nice to keep this log around if possible.

Reply:
fixed